### PR TITLE
New subsection to discuss relation to other systems

### LINKED ISSUE #30: Relationship to other systems
Explain the situations where SSVC is complementary to other vul mgmt systems, and where there is overlap. 
 - [x] CVSS
 - [x] EPSS
 - [x] Tenable VPR
 - [x] https://github.com/varchashva/vPrioritizer

All these systems have seen updates since SSVC v1, make sure the discussion is up to date.

### PR DESCRIPTION
**Pull request by j---**
closes #30

### CODE REVIEW

**Comment by j---** [comment]
I'm not totally sure this is the right place for the subsection, but I do think it should go after all the SSVC decision points are defined. I put it in its own file so it's a bit easier to move if we want to do so.

**Review by ahouseholder — APPROVED** [review]

File: doc/version_1/062_relatedSystems.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+### EPSS
+
+[EPSS](https://www.first.org/epss/) is an "effort for predicting when software vulnerabilities will be exploited."
+EPSS is currently based on a machine-learning classifier and proprietary IDS alert data from Kenna Security.
+While the group has made an effort to make the ML classifier transparent, ML classifiers are not able to provide an intelligible, human-accessible explanation for their behavior [@spring2019ml].
+The use of proprietary training data makes the system less transparent.
+
+EPSS could be used to inform the [*Exploitation*](#exploitation) decision point.
+Currently, [*Exploitation*](#exploitation) focuses on the observable state of the world at the time of the SSVC decision.
+EPSS is about predicting if a transition will occur from the SSVC state of [*none*](#xploitation) to [*proof of concept*](#exploitation).
```
Comment (zmanion) [inline doc/version_1/062_relatedSystems.md:74]: I though EPSS was transition from (none or PoC) to active?

Comment (j---) [reply doc/version_1/062_relatedSystems.md:74]: Ah, you're right.

File: doc/version_1/062_relatedSystems.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
+
+## Related Vulnerability Management Systems
+
+There are several other bodies of work that are used in practice to assist vulnerability managers in making decisions.
+Three relevant systems are CVSS [@cvss_v3-1], EPSS [@jacobs2019exploit], and Tenable's Vulnerability Priority Rating ([VPR](https://www.tenable.com/blog/what-is-vpr-and-how-is-it-different-from-cvss)).
+There are other systems derived from CVSS, such as RVSS for robots and MITRE's effort to adapt CVSS to medical devices.
```
Comment (zmanion) [inline doc/version_1/062_relatedSystems.md:6]: Can we include a reference at least and possibly a bit of text for the medical device CVSS rubric in the CVSS-derived section? Maybe just references for the rubric and RSS?

Comment (zmanion) [reply doc/version_1/062_relatedSystems.md:6]: https://www.mitre.org/publications/technical-papers/rubric-for-applying-cvss-to-medical-devices

Comment (zmanion) [reply doc/version_1/062_relatedSystems.md:6]: or https://www.mitre.org/sites/default/files/publications/pr-18-2208-rubric-for-applying-cvss-to-medical-devices.pdf

Comment (zmanion) [reply doc/version_1/062_relatedSystems.md:6]: https://arxiv.org/abs/1807.10357

**Review by zmanion — APPROVED** [review]

**Pull request merged**